R/pkg/R/context.R

```diff
@@ -25,6 +25,22 @@ getMinPartitions <- function(sc, minPartitions) {
   as.integer(minPartitions)
 }
 
+#' Total number of CPU cores of all executors registered in the cluster at the moment.
+#'
+#' @param sc SparkContext to use
+#' @return current number of cores in the cluster.
+numCores <- function(sc) {
+  callJMethod(sc, "numCores")
+}
+
+#' Total number of executors registered in the cluster at the moment.
+#'
+#' @param sc SparkContext to use
+#' @return current number of executors in the cluster.
+numExecutors <- function(sc) {
+  callJMethod(sc, "numExecutors")
+}
+
```

Comment:
actually, all sparkContext methods (ie. parameter has `sc`) are internal/non-public/deprecated. 
see `spark.addFile`

Reply:
Thank you for pointing me out the example of `spark.addFile`. I changed `spark.numCores` and `spark.numExecutors` in the same way.